Libraries/LibMarkdown/List.cpp

```diff
@@ -58,7 +58,7 @@ String List::render_for_terminal() const
             builder.appendf("%d. ", ++i);
         else
             builder.append("* ");
-        builder.append(item.render_for_terminal());
+        builder.append(item.render_for_terminal(view_width));
```

Comment:
This seems wrong — we've already taken up some of that horizontal space with the `* ` or the `1. `, yet you're passing the full width.

Reply:
Ah, I forgot to fix those.
my bad!